jobs/JEVS_RTOFS_STATS

```diff
@@ -30,6 +30,15 @@ export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
 mkdir -p $DATA
 cd $DATA
 
+####################################
+# Define NET/RUN variables
+####################################
+export NET=${NET:-evs}
+export STEP=${STEP:-prep}
+export COMPONENT=${COMPONENT:-rtofs}
+export RUNsmall=${RUNsmall:-ocean}
```

Comment:
@SamiraArdani-NOAA Please excuse my potential ignorance, but can $RUNsmall be shortened to just $RUN? Is just $RUN used somewhere else that prevents this? Is there a $RUNfinal, etc? Just curious. 

CC @malloryprow

Reply:
I see that RUN for the rtofs stats jobs gets defined in the drivers scripts. Example, dev/drivers/scripts/stats/rtofs/jevs_rtofs_ndbc_grid2obs_stats.sh `RUN=ndbc`.

Reply:
@AliciaBentley-NOAA @malloryprow : Thanks for the comment. $RUN has been used in other scripts.

Reply:
Thanks, @SamiraArdani-NOAA and @malloryprow ! I don't know if `export RUNsmall` necessarily needs to be removed here, but I do think that an `export RUN` line needs to be added that will get overwritten by what RUN is being set to in the dev drivers. I hope that makes sense!

Reply:
If it is being used elsewhere then it will need to be defined in the J-Job too. I don't see any `export RUN=${RUN:-ndbc}` or something similar in there. Can we get that into this J-Job?